src/metabase/api/action.clj

```diff
@@ -93,10 +93,10 @@
   api/generic-204-no-content)
 
 (api/defendpoint POST "/"
-  [action database]
+  [:as {action :body} database]
   (when database
     (do-check-actions-enabled database nil))
-  (when (not= :http (:type action))
+  (when (not= "http" (:type action))
```

Comment:
The action is in the body, and the `$.body.action.type` is a string, so these changes look good to me. Thanks!